docs/release.md

```diff
@@ -0,0 +1,167 @@
+```
+
+When this command is run the following steps will be executed in order
+to produce a release.
+
+1. **Generate the next release version number**
+
+    - CLI will use the last release's version number and the version type
+    to generate the next version number
+
+2. **Prep conformance resources for release**
+
+    - The `version` attribute on all conformance resources will be populated
+      with the new version number.
+    - The `publisher` attribute on all conformance resources with the
+      Kids First canonical URL will be populated with value in
+      `ImplementationGuide.publisher`
+    - The `contact` attribute on all conformance resources with the
+      Kids First canonical URL will be populated with appropriate information
+    - The `date` attribute on all conformance resources will be populated with
+      the current date.
+
+3. **Produce the NPM package for the model**
+
+    - Run the IG publisher to produce the NPM package for the model
+    - Create a tarball of the NPM package
+
+3. **Update `CHANGELOG.md` with release notes**
+
+    - Generate release notes using titles of Git Pull Requests merged into
+      `master` branch since last release
+
+4. **Create a release-version branch and Pull Request**
+
+    - Create a release-version branch off of `master`: `release-<version number>`
+    - Create a Pull Request to merge this branch into `release` branch
+    - Set the PR description to the release notes markdown
+
+5. **Team must review and approve the Pull Request**
+
+    - The only FHIR model resource changes to review at this point will be
+      those made in [Release Process step 2](#Release-Process).
+
+    - The team must also review the release note changes in `CHANGELOG.md`.
+
+6. **Create Github Release**
+    - On merge of release-version branch into `release` branch,
+      create the Github release
+    - Attach the NPM package tarball as a release artifact
+
+7. **Publish the IG site**
+    - On merge of release-version branch into `release` branch,
+      trigger the IG site publishing process so that IG Github pages site is
+      updated with the released conformance resources.
+
+
+## Deployment
+
+### Steps
+
+1. On a merge into the release branch the following happens:
+    - FHIR model's onformance resources and example resources are deployed
+      to an integration test FHIR server.
+    - Run any data migration tests on the integration test FHIR server
+3. If test deployment fails for any reason, go back to step 1 of
+   [Pre-Release](#Pre-Release)
+3. If the deployment to the integration test server is successful:
+    - Deploy the conformance resources to the production server
+    - Run any data migrations
+4. Deployment to production could automatically be triggered on successful
+   deployment to integration test server OR it could be a separate process
+   triggered manually when we are ready to update the model in the production
+   server.
```

Comment:
The expected sequence of events is unclear from this. You have "Team must review and approve the Pull Request" _inside_ of a section that starts with "When this command is run the following steps will be executed", and then the next entry is "Create Github Release". Who does?